packages/plugins/scripts-runner/src/index.ts

```diff
@@ -0,0 +1,178 @@
+import { eachSeries } from 'async';
+import { promises as fsPromises, stat, Stats } from 'fs';
+import path from 'path';
+import { Embark, Callback } from 'embark-core';
+import { Logger } from 'embark-logger';
+import { dappPath } from 'embark-utils';
+import Web3 from "web3";
+
+export enum ScriptsRunnerCommand {
+  Execute = 'scripts-runner:execute'
+}
```

Comment:
We could add the other individual commands `ExecuteSingle` and `ExecuteAll` for more fine control. Either way, `Execute` will take care of delegating to the right sub command.